content/en/continuous_delivery/deployments/ciproviders.md

```diff
@@ -90,6 +92,36 @@ export DD_SITE={{< region-param key="dd_site" >}}
 datadog-ci deployment mark --env prod --tags team:backend --tags reason:scheduled
 ```
 
+## Mark GitHub Actions jobs as deployments
```

Comment:
This is very similar to the section in custom_tags_and_metrics (the deployment command is using the tag command under the hood) but I preferred to duplicate it as the example is clearer (more deployment-related) and otherwise it might be confusing for users to read the "custom tags" documentation for the deployment command